_includes/documentation_eg.html

```diff
@@ -628,8 +628,10 @@ <h4 class="example-section-title" id="layout-options">
 {% endif %}
 
 {% if page.dash_url %}
-<h1>Dash Example</h1>
+<a href="#dash-example">Dash Example</a>
```

Comment:
This is too small now, it needs to be a link inside header.
```suggestion
<h1><a href="#dash-example">Dash Example</a></h1>
```